ocaml/testsuite/tests/typing-unique/overwriting_lift_constants_bug.ml

```diff
@@ -17,6 +19,24 @@ let fconstant_lift b =
   let unique_ p = { x = 1.0; y = 2.0; z = 3.0 } in
   if b then p else overwrite_ p with { x = 2.0 }
 
+type mpoint = { dim : int; x : float#; y : float#; z : float# }
+
+let mconstant_lift b =
+  let unique_ p = { dim = 3; x = #1.0; y = #2.0; z = #3.0 } in
```

Comment:
This isn't quite a mixed block: immediates (such as `int`) fit in either section of the block. Better to use something that's actually a pointer.